docs/Tutorials/Quorum-Dev-Quickstart/Getting-Started.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Quorum Dev Quickstart network setup quickstart command line tool
+---
+
+# Quorum Dev Quickstart
+
+[Quorum Dev Quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) is a command line tool that allows
+users to set up a development GoQuorum network on their local machine in less than two minutes.
+The quickstart is written in Javascript and designed to be run as a global npm module from the command line.
+
+![Quorum Dev Quickstart terminal demo](../../images/quickstart/quorum-dev-quickstart.gif)
```

Comment:
This file doesn't exist in the documentation. Did you mean to add it here? Otherwise this should be deleted.

Reply:
Yep, making an animated gif of this today :)

Reply:
Should be good now for review @alexandratran :) Thank you